src/ui/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,186 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Mozilla.VPN 1.0
+import components 0.1
+import components.forms 0.1
+
+Item {
+    id: root
+
+    property string _menuTitle: VPNl18n.TipsAndTricksSettingsEntryLabel
+    readonly property int tabletSize: 600 //Tablets get a different layout (2 columns for tutorials, 3 columns for guides)
+
+    objectName: "settingsTipsAndTricks"
+
+    VPNFlickable {
+        id: vpnFlickable
+
+        property int screenWidth: width
+
+        anchors.fill: parent
+        anchors.topMargin: VPNTheme.theme.menuHeight
+
+        flickContentHeight: layout.implicitHeight + layout.anchors.topMargin
+        interactive: flickContentHeight > height
+
+        Column {
+            id: layout
+            anchors.top: parent.top
+            anchors.left: parent.left
+            anchors.right: parent.right
+            anchors.topMargin: VPNTheme.theme.vSpacing
+            anchors.leftMargin: VPNTheme.theme.windowMargin
+            anchors.rightMargin: VPNTheme.theme.windowMargin
+
+            spacing: 0
+
+            //superfluous columnlayout fixes binding loop
+            ColumnLayout {
+                anchors.left: parent.left
+                anchors.right: parent.right
+
+                spacing: 0
+
+                Flow {
+                    Layout.fillWidth: true
+                    spacing: VPNTheme.theme.vSpacingSmall
+
+                    Loader {
+                        id: firstTutorialLoader
+                        property variant firstTutorial: VPNTutorial.get(0)
+
+                        height: 144
+                        width: vpnFlickable.screenWidth < root.tabletSize ? parent.width : (parent.width - parent.spacing) / 2
+
+                        active: firstTutorial
+                        visible: active
+
+                        sourceComponent: VPNTutorialCard {
+                            width: parent.width
+                            height: parent.height
+
+                            imageSrc: firstTutorial.image
+                            imageBgColor: firstTutorial.imageBgColor
+                            title: VPNl18n[firstTutorial.titleId]
+                            description: VPNl18n[firstTutorial.subtitleId]
+                        }
+                    }
+
+                    VPNTutorialCard {
+                        width: vpnFlickable.screenWidth < root.tabletSize ? parent.width : (parent.width - parent.spacing) / 2
```

Comment:
Any reason to not specify the width/height in VPNTutorialCard.qml ?

Reply:
In this specific case, the width of the tutorial cards are sometimes reliant on ancestor components which seems like bad practice to refer to from inside the component file. 

The height is always the same for all tutorial cards here, but I guess in the off chance that these cards are used in some other view at some point with a different height, it may be weird to have 'default' height 🤷